news/3741.bugfix

```diff
@@ -0,0 +1 @@
+Fix ``pip uninstall`` when ``easy-install.pth`` lacks a trailing newline.
```

Comment:
Single back-ticks?

Reply:
Convention seems to be double-backticks in all the other files, since this is rest. Should that change?

Reply:
oh, it's rst. nvm me. :)